### DISCUSSION TITLE
Can't run my project for TvOs, Android Tv, Web (Basically every platform)

### DISCUSSION THREAD

**Discussion by Sifat-16** [pavjacko/renative#1326] [Q&A] [answered]
I am following the documentation properly and trying every possible combinations to  run the project on my mac, but it's not working. There are always some errors. Can you please give me a proper instructions of how to run the project? TvOs is my first priority.
Thank you.

**Comment by m7mdmakkeh** [comment]
Same, basic project couldn't be run

**Comment by pauliusguzas** [comment]
@Sifat-16 @m7mdmakkeh hello, please use 1.0.0-rc.12 and if any issues arise create them as a ticket

**Reply by Parvinderjit** [reply]
When Are you planning to stabilise it ?
I mean there are so many RC releases and there is not stable version that supports the RN 70+

**Comment by Fyb3roptik** [comment]
I am having the exact same issue. Brand new project, trying latest 1.0.0-rc.13 and it didnt install any engine! After figuring out how to update the json file with the correct engine, when I go to build an emulator for firetv, it simply just hangs. Android studio is launched...nothing ever happens!

Does this even work for anyone else?! I followed the docs to the letter and I cannot for the life of me get this to even launch an emulator. Is there a Discord or something where I can get some help?

**Comment by pauliusguzas** [comment, marked as answer]
@Fyb3roptik @Parvinderjit @m7mdmakkeh @Sifat-16 Sorry for late reply, Renative has been live with stable release for some time now, latest release from npm shoudn't have any big issues. If you encounter anything please open a ticket. Thank you for understanding

**Comment by Fyb3roptik** [comment]
> @Fyb3roptik @Parvinderjit @m7mdmakkeh @Sifat-16 Sorry for late reply, Renative has been live with stable release for some time now, latest release from npm shoudn't have any big issues. If you encounter anything please open a ticket. Thank you for understanding

What do you think this is genius?

Literally couldn't get even the simplest thing to compile. None of it works. I've seen many many posts about this so I moved on to something better.
